python/cuml/test/test_coordinate_descent.py

```diff
@@ -112,13 +110,6 @@ def test_elastic_net(datatype, X_type, lr, algorithm,
     X_train = np.asarray(X[0:train_rows, :], dtype=datatype)
     y_train = np.asarray(y[0:train_rows, ], dtype=datatype)
 
-    if nrows != 500000:
-        elastic_sk = ElasticNet(alpha=np.array([0.1]), fit_intercept=True,
-                                normalize=False, max_iter=1000,
-                                selection=algorithm, tol=1e-10)
-
-        elastic_sk.fit(X_train, y_train)
-
     elastic_cu = cuElasticNet(alpha=np.array([0.1]), fit_intercept=True,
```

Comment:
Can we test elastic-net with different alpha values as well? Like 0.2 and 0.7.

Reply:
done!